Framework/PythonInterface/mantid/kernel/src/Converters/CloneToNumpy.cpp

```diff
@@ -65,6 +81,7 @@ PyObject *cloneND(const ElementType *carray, const int ndims,
   }
   void *arrayData = PyArray_DATA(nparray);
```

Comment:
Is there a reason we can't just check for the zero-length here and return rather than requiring a separate function?